app/src/main/java/org/p2p/wallet/restore/ui/keys/SecretKeyFragment.kt

```diff
@@ -86,8 +86,6 @@ class SecretKeyFragment :
             termsAndConditionsTextView.text = buildTermsAndPrivacyText()
             termsAndConditionsTextView.movementMethod = LinkMovementMethod.getInstance()
         }
-
-        setButtonEnabled(phraseAdapter.itemCount != 0)
```

Comment:
We need to enable the button if we navigated back to this screen. 
I think we shouldn't delete it but add check for `phraseAdapter.itemCount == 12 or 24`